cas-server-support-osf/src/main/java/org/scribe/builder/api/OrcidApi20.java

```diff
@@ -35,32 +35,19 @@
     private static final String AUTH_URL = "https://www.orcid.org/oauth/authorize";
 
     /** The token exchange url. */
-    private static final String TOKEN_URL = "https://%s.orcid.org/oauth/token";
-
-    /** The member flag. */
-    private final Boolean member;
+    private static final String TOKEN_URL = "https://orcid.org/oauth/token";
```

Comment:
The current class structures makes it hard to make this a settings that we can load via bean init. Similar to the original code, I hard-code the new token URL here.